### PR TITLE
[hi] Localized concepts/containers/container-environment.md

### PR DESCRIPTION
**Pull request by aj11anuj**

### CODE REVIEW

File: content/hi/docs/concepts/containers/container-environment.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* एक फाइल सिस्टम, जो एक [छवि](/hi/docs/concepts/containers/images/) और एक या अधिक
+  [वॉल्यूम](/hi/docs/concepts/storage/volumes/) का एक मेल है।
+* कंटेनर के बारे में जानकारी।
+* क्लस्टर के अन्य ऑब्जेक्ट्स के बारे में जानकारी।
+
+
+### कंटेनर की जानकारी
+
+एक कंटेनर का *होस्टनेम* उस पॉड का नाम है, जिसमें कंटेनर चल रहा है।
+यह 'होस्टनेम' कमांड के माध्यम से उपलब्ध है या
```
Comment (kundan2707) [inline content/hi/docs/concepts/containers/container-environment.md:30]: command like hostname and gethostname should not be translated

File: content/hi/docs/concepts/containers/container-environment.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+['गेटहोस्टनेम'](https://man7.org/linux/man-pages/man2/gethostname.2.html)
+एलआईबीसी में फ़ंक्शन कॉल के माध्यम से।
+
+पॉड नाम और नेमस्पेस, वातावरण के वेरिएबल के रूप में उपलब्ध हैं,
```
Comment (kundan2707) [inline content/hi/docs/concepts/containers/container-environment.md:34]: > 
```suggestion
पॉड नाम और नेमस्पेस, वातावरण के वेरिएबल के रूप में 
[downward API](/hi/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) की सहायता से उपलब्ध हैं |
```

File: content/hi/docs/concepts/containers/container-environment.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+*foo* नामक सेवा, जो *bar* नामक कंटेनर को दर्शा रही हैै, उनके लिए
+निम्न लिखित वेरिएबल्स परिभाषित किए गए हैं:
+
+```shell
+FOO_SERVICE_HOST=<the host the service is running on>
+FOO_SERVICE_PORT=<the port the service is running on>
+```
+
+सेवाओं के पास आईपी एड्रेस होते है, जो की डीएनएस के माध्यम से कंटेनर के लिए उपलब्ध होते हैं,
+यदि [DNS addon](https://releases.k8s.io/v{{< skew currentPatchVersion >}}/cluster/addon/dns/) चालू किया गया है।
```
Comment (kundan2707) [inline content/hi/docs/concepts/containers/container-environment.md:53]:
```suggestion
यदि [DNS addon](https://releases.k8s.io/v{{< skew currentPatchVersion >}}/cluster/addon/dns/) चालू किया गया है।
सेवाओं के पास समर्पित आईपी एड्रेस डीएनएस के माध्यम से कंटेनर के लिए उपलब्ध होते हैं ।
```

**Comment by divya-mohan0209** [comment]
@aj11anuj Since this PR has had no activity for a long time, I am closing out the PR. If and when you have the bandwidth, please feel free to check Kundan's suggestion and we can resume work.

/close

**Pull request closed without merging**